modules/assim.batch/R/pda.define.llik.R

```diff
@@ -71,17 +71,26 @@ pda.calc.error <-function(settings, con, model_out, run.id, inputs, bias.terms){
         
         resid <- abs(model_out[[k]] - inputs[[k]]$obs)
         pos <- (model_out[[k]] >= 0)
-        SS <- c(dexp(resid[pos],
-                     1 / (inputs[[k]]$par[1] + (inputs[[k]]$par[2] * 
-                                                  sqrt(inputs[[k]]$n_eff/inputs[[k]]$n) * 
-                                                  model_out[[k]][pos])), log = TRUE),
-                dexp(resid[!pos],
-                     1 / (inputs[[k]]$par[1] + (inputs[[k]]$par[3] * 
-                            sqrt(inputs[[k]]$n_eff/inputs[[k]]$n) * 
-                            model_out[[k]][!pos])), log = TRUE))
+        # SS <- c(dexp(resid[pos],
+        #              1 / (inputs[[k]]$par[1] + (inputs[[k]]$par[2] * 
+        #                                           sqrt(inputs[[k]]$n_eff/inputs[[k]]$n) * 
+        #                                           model_out[[k]][pos])), log = TRUE),
+        #         dexp(resid[!pos],
+        #              1 / (inputs[[k]]$par[1] + (inputs[[k]]$par[3] * 
+        #                     sqrt(inputs[[k]]$n_eff/inputs[[k]]$n) * 
+        #                     model_out[[k]][!pos])), log = TRUE))
+        # 
+        # pda.errors[[k]] <- sum(SS, na.rm = TRUE) 
+        # SSdb[[k]] <- sum(SS, na.rm = TRUE) 
         
-        pda.errors[[k]] <- sum(SS, na.rm = TRUE) 
-        SSdb[[k]] <- sum(SS, na.rm = TRUE) 
+        beta_p <- (inputs[[k]]$par[1] + inputs[[k]]$par[2] * model_out[[k]][pos]* sqrt(inputs[[k]]$n/inputs[[k]]$n_eff) ) 
+        beta_n <- (inputs[[k]]$par[1] + inputs[[k]]$par[3] * model_out[[k]][!pos]* sqrt(inputs[[k]]$n/inputs[[k]]$n_eff)) 
+        suppressWarnings(if(length(beta_n) == 0) beta_n <- 0)
+        SS_p <- - (inputs[[k]]$n_eff/inputs[[k]]$n) * log(beta_p) - resid[[1]][pos]/beta_p
+        SS_n <- - (inputs[[k]]$n_eff/inputs[[k]]$n) * log(beta_n) - resid[[1]][!pos]/beta_n
```

Comment:
@mdietze is this correct? After @ashiklom corrected the likelihood with `n_eff` I thought this might be relevant here as well, weighing the likelihood down for the flux data